mypyc/test-data/run-strings.test

```diff
@@ -342,6 +342,12 @@ def test_format_method_args() -> None:
     assert format_kwargs(x=10, y=2, z=1) == 'c10d2'
     assert format_kwargs_self(x=10, y=2, z=1) == "{'x': 10, 'y': 2, 'z': 1}"
 
+def test_format_method_different_kind() -> None:
+    s1 = "Literal['😀']"
+    assert 'Revealed type is {}'.format(s1) == "Revealed type is Literal['😀']"
+    s2 = "Revealed type is"
+    assert "{} Literal['😀']".format(s2) == "Revealed type is Literal['😀']"
```

Comment:
It would be useful to try all combinations of 1, 2 and 4 byte wide characters, such as 2 byte format string and 4 byte argument, etc.